app/controllers/uploads_controller.rb

```diff
@@ -11,7 +11,7 @@ def new
   # POST /apos/:apo_id/uploads
   def create
     directory_name = Time.zone.now.strftime('%Y_%m_%d_%H_%M_%S_%L')
-    output_directory = File.join(Settings.bulk_metadata.directory, params[:druid], directory_name)
+    output_directory = File.join(Settings.bulk_metadata.directory, params[:apo_id], directory_name)
```

Comment:
what if you're working with multiple druids for the same APO?

Reply:
I'm not following.  Can you be more specific on how you would interact with Argo in order to do that?

Reply:
I wasn't thinking big picture - I was looking at this code change and wondering if we would ever have a problem because we are using the apo id for a directory name rather than the actual desired druid.  I don't know if the directory is exposed to the user, or how the user actions get to this code.

Reply:
The parameter that was `druid` was always holding the same value as `apo_id`